### PR TITLE
Make sure the displayed PRs are unic

### PR DESCRIPTION
**Pull request by cleicar**
This will address issue: #59

### CODE REVIEW

**Review by FionaDL — APPROVED** [review]
@cleicar looks good to me.

File: app/models/user.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class User < ApplicationRecord
-
```
Comment (bronzdoc) [inline app/models/user.rb:2]: can you remove this whitespace?

File: app/models/user.rb [hunk 2]
```diff
@@ -11,7 +11,7 @@ class User < ApplicationRecord
 
   has_many :pivotal_stories_owners
   has_many :pivotal_stories, through: :pivotal_stories_owners
-
```
Comment (bronzdoc) [inline app/models/user.rb:14]: Can you remove this whitespace?

**Review by bronzdoc — CHANGES_REQUESTED** [review]
Looks good with a test 👍

**Comment by cleicar** [comment]
Thanks for your review @bronzdoc. I just pushed the changes.

**Comment by bronzdoc** [comment]
Thanks @cleicar! This change needs a test and we good to go 💪

**Comment by cleicar** [comment]
@bronzdoc done!

**Review by bronzdoc — APPROVED** [review]
@ABizzinotto this is ready for QA

**Comment by ABizzinotto** [comment]
@cleicar good to merge!

**Pull request merged**